## /bin/sh -e simulate dpatch for prepend_chlog.py
## Build
## Makefile_clean.dpatch by Kilian Krause <kilian@debian.org>
## DP: Unsuccessful cleanup is not an error. Fixes #393659.

Index: asterisk-1.2.27/Makefile
===================================================================
--- asterisk-1.2.27.orig/Makefile	2008-03-19 11:33:39.000000000 +0100
+++ asterisk-1.2.27/Makefile	2008-03-19 11:33:41.000000000 +0100
@@ -537,7 +537,7 @@
 	for x in $(SUBDIRS); do $(MAKE) -C $$x || exit 1 ; done
 
 clean:
-	for x in $(SUBDIRS); do $(MAKE) -C $$x clean || exit 1 ; done
+	for x in $(SUBDIRS); do $(MAKE) -C $$x clean || true ; done
 	rm -f *.o *.so asterisk .depend
 	rm -f defaults.h
 	rm -f include/asterisk/build.h
